test/avatar.test.js

```diff
@@ -0,0 +1,35 @@
+import createCore from './helpers/core';
+import getAccount from './helpers/account';
+import { deploySafeAndToken } from './helpers/transactions';
+import { mockApiAvatarUpload, mockApiAvatarDelete } from './helpers/mocks';
+
+let account;
+let core;
+
+beforeAll(async () => {
+  core = createCore();
+});
+
+describe('Avatar - upload and delete', () => {
+  beforeAll(async () => {
+    account = getAccount(1);
+    // The Safe must be deployed and signedup to the Hub before trying to delete the user entry
+    await deploySafeAndToken(core, account);
+  });
+
+  describe('when a user wants to upload an image, and then remove it', () => {
```

Comment:
Should we test trying to remove an image that is in use or that doesn't make sense to mock and is tested in api right?

Reply:
That should be tested in the API.